src/Microsoft.ML.CodeGenerator/Templates/Console/PredictProject.tt

```diff
@@ -20,6 +20,12 @@
 <#}#>
 <# if (IncludeFastTreePackage){ #>
     <PackageReference Include="Microsoft.ML.FastTree" Version="1.3.1" />
+<#}#>
+<# if (IncludeImageTransformerPackage){ #>
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.4.0" />
```

Comment:
This code isn't going to work until we ship `1.4.0`.

We may need a way to update these numbers during the build.

Reply:
Is there a cleaner way to include the build numbers as variables as we do elsewhere?

Reply:
@eerhardt this is how code generator is structured, they hardcode version numbers. May be they can make this better as a separate change.

Reply:
Would it be possible to change the other package version numbers to next releases' numbers than? Otherwise the codegen (if released like current) will pull in old packages. This is step needed before release anyway.

Reply:
`Version="1.4.0"` doesn't exist yet. So the users project isn't going to build until we publish `1.4.0`.